modules/nixos/general/keyboard.nix

```diff
@@ -49,11 +49,11 @@ with lib;
   };
 
   config = {
-    boot.earlyVconsoleSetup = cfg.enableAtBoot;
-    i18n.consoleKeyMap = let layout = layouts."${builtins.head cfg.layouts}"; in
-                         if layout.variant == ""
-                         then layout.layout
-                         else (if layout.variant == "bepo" then "fr-bepo" else layout.variant);
+    console.earlySetup = cfg.enableAtBoot;
+    console.keyMap = let layout = layouts."${builtins.head cfg.layouts}"; in
```

Comment:
Are these working on 19.03 or are these only 20.03?

Reply:
They are only 20.03

Reply:
Is it a blocker?

Reply:
It is for keeping 19.03. So you need to either split on the version or just remove 19.03 and make 20.03 default. I don't mind the latter.

Reply:
I will remove 19.09